src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py

```diff
@@ -316,6 +316,9 @@ def _raw_input(self, index):
     def num_of_outputs(self):
         return len(self.outputs())
 
+    def num_of_users(self):
+        return len(self.pt_module.users)
```

Comment:
Yes, this could fail in that case even though I haven't observed such a usage yet. So, I updated this implementation to return the size of the output list if the user nodes are getitems.